### PR TITLE
Translate "Simple Values" to Spanish

### PR DESCRIPTION
**Pull request by gnandretta**
I need to do a final review, but the content is there ✨

### CODE REVIEW

**Comment by allentiak** [comment]
Adresses #13

File: es/outline/simple_values.md [hunk 1]
```diff
@@ -0,0 +1,336 @@
...
+
+<section ng-controller="NarrativeController">
+### Booleanos y nil
+{: .slide_title .slide}
+
+#### <button class="link" ng-bind-html="details" ng-model="block31" ng-click="block31=!block31"></button>
+
+> Un booleano indica que algo es verdadero o es falso y en Clojure los
+> escribimos como `true` o `false`, respectivamente. Cuando programamos, es
+> común hacer preguntas del tipo ""¿Hoy es el cumpleaños de tal persona?" o
```
Comment (nberger) [inline es/outline/simple_values.md]: Nit: doble comillas dobles :)

File: es/outline/simple_values.md [hunk 2]
```diff
@@ -0,0 +1,336 @@
...
+<section ng-controller="NarrativeController">
+#### Fracciones <button class="link" ng-bind-html="details" ng-model="block101" ng-click="block101=!block101"></button>
+
+> Por último, tenemos las fracciones. Una computadora no puede representar
+> perfectamente a todos los números decimales y es posible que utilice una misma
+> representación para más de un número. Esto no ocurre con las fracciones que
+> son siempre exactas. Las podemos reconocer por la barra inclinada "/":
+{: ng-show="block101" .description}
+
+> Es importante tener en cuenta que, al igual que en la clases de matemática, el [denominador](https://es.wikipedia.org/wiki/Fracci%C3%B3n) de una fracción no puede ser igual a `0`.
```
Comment (nberger) [inline es/outline/simple_values.md]: s/clases/clase

File: es/outline/simple_values.md [hunk 3]
```diff
@@ -0,0 +1,336 @@
...
+
+{% comment %}
+
+:star2: El enlace de abajo es para ser usado en la presentación. Podés volver a [README.md](../README.md). :star2:
+
+{% endcomment %}
+
+<section>
+Podés volver al <a href="javascript:;" onClick="Reveal.slide(1);">principio de la presentación</a>,
+or al [índice del programa](/curriculum/#/1).
```
Comment (nberger) [inline es/outline/simple_values.md]: s/or/o :)
o mejor aún copiar del final de https://github.com/ClojureBridgeBA/curriculum/blob/gh-pages-es/es/outline/data_structures.md por consistencia

**Comment by nberger** [comment]
:+1: por mi parte, excepto los comentarios menores que hice

**Comment by gnandretta** [comment]
@nberger thanks for the corrections, the last commit (af158e2) should address them.

**Comment by nberger** [comment]
Great! Thanks @gnandretta! :ship:

**Pull request merged**